packages/llvm.rb

```diff
@@ -39,8 +39,8 @@ class Llvm < Package
   when 'aarch64','armv7l'
     #LLVM_TARGETS_TO_BUILD = 'ARM;AArch64;AMDGPU'
     LLVM_TARGETS_TO_BUILD = 'all'
-    @ARCH_C_FLAGS = "-fPIC -march=armv7-a -mfloat-abi=hard -ccc-gcc-name #{LLVM_DEFAULT_TARGET_TRIPLE}"
-    @ARCH_CXX_FLAGS = "-fPIC -march=armv7-a -mfloat-abi=hard -ccc-gcc-name #{LLVM_DEFAULT_TARGET_TRIPLE}"
+    #@ARCH_C_FLAGS = "-fPIC -march=armv7-a -mfloat-abi=hard -ccc-gcc-name #{LLVM_DEFAULT_TARGET_TRIPLE}"
+    #@ARCH_CXX_FLAGS = "-fPIC -march=armv7-a -mfloat-abi=hard -ccc-gcc-name #{LLVM_DEFAULT_TARGET_TRIPLE}"
```

Comment:
Add two lines below ? (as it may cause errors in setting the `@ARCH_C_LTO_FLAGS` variable below if these variables not set)
```ruby
@ARCH_C_FLAGS = ' '
@ARCH_CXX_FLAGS = ' '
```

Reply:
This is just a quick fix to get rid of the error.  Since this is a core package, users who try to install fresh or upgrade will experience a crash...not a good experience.

Reply:
I'm confused. Why does ``` @ARCH_C_FLAGS = "-fPIC -march=armv7-a -mfloat-abi=hard -ccc-gcc-name #{LLVM_DEFAULT_TARGET_TRIPLE}"``` not work?

Isn't this literally how we specify ```@_ver = 'something'``` ?

Reply:
Ah I see. @supechicken 's fix appears to be the correct one.

Reply:
The actual correct solution is to replace ```#{LLVM_DEFAULT_TARGET_TRIPLE}``` with ```#{CREW_BUILD}```